<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArchivingViewer</class>
 <widget class="QMainWindow" name="ArchivingViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>622</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>941</width>
    <height>622</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Archiving Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons/icons.qrc">
    <normaloff>:/appicon/archive.png</normaloff>:/appicon/archive.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="date_controls">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>901</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QDateTimeEdit" name="begin_date">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>194</width>
       <height>27</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy HH:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="end_date">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>194</width>
       <height>27</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy HH:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="choose_be">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Begin and end date</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="choose_per">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Period of time</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>15</y>
       <width>87</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Begin:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>45</y>
       <width>87</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>End:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="end_now">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>45</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Now</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>75</y>
       <width>87</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Time span:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="refresh">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>170</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string> Refresh plot</string>
     </property>
     <property name="icon">
      <iconset resource="../Icons/icons.qrc">
       <normaloff>:/buttons/refresh.png</normaloff>:/buttons/refresh.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QSpinBox" name="span_days">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="suffix">
      <string notr="true">d</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="span_hours">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>70</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="suffix">
      <string notr="true">h</string>
     </property>
    </widget>
    <widget class="QComboBox" name="span_minutes">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>51</width>
       <height>27</height>
      </rect>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="choose_last">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Last</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="smooth_len">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>50</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="smooth_en">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Smooting</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>580</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QWidget" name="splitter_area" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>901</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
